apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -85,21 +85,21 @@ public SpannerCommand()
         /// <inheritdoc />
         public override string CommandText
         {
-            get { return SpannerCommandTextBuilder?.ToString() ?? string.Empty; }
-            set { SpannerCommandTextBuilder = SpannerCommandTextBuilder.FromCommandText(value); }
+            get => SpannerCommandTextBuilder?.ToString() ?? string.Empty;
```

Comment:
is there something wrong with string.Empty?  not sure I understand the problem.
If its a coding standard thing, I'm fine with it, but we should start collecting what our standards our.  maybe this is already documented?